utils/buildcli/buildcli/clang.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+
+from collections import namedtuple
+import platform
+import subprocess
+
+from . import xcrun
+from .which import which
+
+# host_clang() prefers versions of Clang in this order.
+# Note that this array must be updated when new, major versions of Clang
+# are released.
+_CLANG_SUFFIXES = ['38', '37', '36', '35']
```

Comment:
This should be separate for Linux and FreeBSD.

Or maybe it should include both kinds of suffixes.